_source/_posts/2021-02-26-unity-webgl-playfab-authorization.md

```diff
@@ -0,0 +1,654 @@
+
+The good news is that once the target is defined and the browser interaction decided, the rest of the code should be similar across all platforms. This is thanks to the shared responsibility with a security platform's hosted login experience and a standardized authorization specification. The job of the developer is to securely provide a platform-preferred browser experience for the user and direct them to the OAuth authorize endpoint. Once the user has authenticated, the browser is closed, and an auth code is captured in client code to be exchanged for tokens behind the scenes. With tokens in hand, user authorization can be passed to all other backend cloud platforms; providing a centralized, and secure, identity security design that drastically simplifies the implementation effort.
+
+## Create the Project
+
+First, if you have not already done so, create a new project from `Unity Hub` with the `3D` template.
+
+{% img blog/unity-webgl-playfab-authorization/1_NewProject.png alt:"New Project" width:"800" %}{: .center-image }
+
+Next, configure the Playfab Unity SDK and Editor Extensions using the [Unity Quickstart](https://docs.microsoft.com/en-us/gaming/playfab/sdks/unity3d/quickstart) provided by PlayFab. 
+
+{% img blog/unity-webgl-playfab-authorization/2_Playfab.png alt:"PlayFab" width:"800" %}{: .center-image }
+
+Create a new scene by clicking **File**, **New Scene** and save it in the projects `Assets` > `Scenes` folder.
+
+{% img blog/unity-webgl-playfab-authorization/3_NewScene.png alt:"New Scene" width:"800" %}{: .center-image }
+
+Change the project to targetWebGL by, clicking **File**, **Build Settings**, **WebGL**, and then click **Switch Platform**.
+
+{% img blog/unity-webgl-playfab-authorization/4_BuildSettings.png alt:"Build Settings" width:"800" %}{: .center-image }
+
+> Note: If you did not install WebGL when you installed this copy of the editor, you would be prompted to do so now.
+
+Make sure that your newly created scene is checked in the `Scenes In Build` menu, and then close the `Build Settings` UI. 
+
+> Note: If you do not see your scene, click the **Add Open Scenes** button in the bottom right of the `Scenes In Build` menu. 
+
+Right-click in the `Scene Hierarchy` panel and click **UI**, **Button**.
+
+{% img blog/unity-webgl-playfab-authorization/5_NewButton.png alt:"New Button" width:"800" %}{: .center-image }
+
+This will be the button that triggers the authentication request.
+
+Right click the button in the `Hierarchy` panel,  click **Rename** and name the button `Sign In.`
+
+{% img blog/unity-webgl-playfab-authorization/6_RenameButton.png alt:"Rename Button" width:"800" %}{: .center-image }
```

Comment:
This image renders really big in my preview. I suggest removing `width:"800"` from the `img` tag, or changing the width to the native width of the image.

Reply:
Changed to 400px